src/shared/components/wallet-enclave/WalletEnclave.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component, Fragment } from 'react';
```

Comment:
This file is supposed to be in app/

Reply:
This was not like that in the master branch, are you sure you want me to move the file to app/?

Reply:
On master, it's in the right place.. https://github.com/ipfs-shipyard/nomios-web/tree/master/src/app

Reply:
Nevermind, misunderstood what you were saying